dbcon/ddlpackage/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 INCLUDE_DIRECTORIES( ${ENGINE_COMMON_INCLUDES} )
 
 FIND_PACKAGE(BISON REQUIRED)
-BISON_TARGET(ddl_gram ddl.y ${CMAKE_CURRENT_BINARY_DIR}/ddl-gram.cpp DEFINES_FILE ${CMAKE_CURRENT_BINARY_DIR}/ddl-gram.h COMPILE_FLAGS "-l -d -p ddl")
+BISON_TARGET(ddl_gram ddl.y ${CMAKE_CURRENT_BINARY_DIR}/ddl-gram.cpp
+  DEFINES_FILE ${CMAKE_CURRENT_BINARY_DIR}/ddl-gram.h
+  COMPILE_FLAGS "-l -p ddl --defines=${CMAKE_CURRENT_BINARY_DIR}/ddl-gram.h")
```

Comment:
Curious, did you figure out why your machine was generating a .hpp file?

Reply:
Not, actually. The only relevant thing was a CMake version difference b/c bison is the same.